apiserver/src/error.rs

```diff
@@ -13,6 +13,9 @@ pub enum Error {
     #[snafu(display("Unable to create client: '{}'", source))]
     ClientCreate { source: kube::Error },
 
+    #[snafu(display("Unable to parse HTTP header. Missing '{}'", missing_header))]
```

Comment:
I remember Tom mentioned something like it's not recommended to define different errors in an error file. Do we want to make some adjustments in that case?

Reply:
I think this is a good idea. I've opened #121 to track moving errors to submodules, since these span multiple existing crates. Is it okay to defer fixing that to a future PR?

Reply:
Sure.